# header    Header                  message header
# human     yolo_detector/Event     last received human detector event
# object    apriltag_detector/Event last received object detector event

Header header
yolo_detector/Event human
apriltag_detector/Event object